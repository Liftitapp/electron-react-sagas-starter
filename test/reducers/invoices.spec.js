import invoicesActions from '../../app/reducers/invoices_redux'

describe('actions', () => {
  it('should fetch', () => {
    expect(invoicesActions.invoicesFetch()).toMatchSnapshot()
  })

  it('should be succes', () => {
    expect(invoicesActions.invoicesSuccess()).toMatchSnapshot()
  })
})
