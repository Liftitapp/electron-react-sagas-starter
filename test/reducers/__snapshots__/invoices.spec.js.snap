// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions should be succes 1`] = `
Object {
  "type": "INVOICES_SUCCESS",
}
`;

exports[`actions should fetch 1`] = `
Object {
  "type": "INVOICES_FETCH",
}
`;
